src/rootPages/Designer/ui_common_popupEditMenu.js

```diff
@@ -179,22 +183,20 @@ export default function (AB) {
             // hide our popup before we trigger any other possible UI animation: (like .edit)
             // NOTE: if the UI is animating another component, and we do .hide()
             // while it is in progress, the UI will glitch and give the user whiplash.
-
             var label = itemNode.textContent.trim();
             var option = this._menuOptions.filter((mo) => {
-               return mo.label == label;
+                return mo.label == label;
             })[0];
-
-            // NOTE: .hide() the popup before .trigger() so we don't auto
-            // close any popups that get triggered.
-            this.hide();
-
             if (option) {
-               // this._logic.callbacks.onClick(option.command);
-               this.trigger(option.command);
+                if (this.onClick) {
```

Comment:
I think I was trying to avoid fetching which item was selected from the dom later: on the interface list level, when it receives a call from the popup form. It currently works, but seems inefficient. 
I removed this pass in order to get it to match the precedent though